### ISSUE TITLE
eBOT and Websocket not online

### ISSUE THREAD

**Issue by xe1os** [Flegma/ebot-cs2-install-script#2]
Hi Flegma,

I didn't find any errors during the script execution.

But both ebot and websocket are showing loading in the webpanel.

![image](https://github.com/Flegma/ebot-cs2-install-script/assets/39219862/6814e55f-712c-4b1b-9ac9-c778daf0a9dc)

**Comment by Flegma** [comment]
Can you try and enter screens that is running the app (probably called ebot-cs2-app for you) and let me know what is the error?

**Comment by xe1os** [comment]
> Can you try and enter screens that is running the app (probably called ebot-cs2-app for you) and let me know what is the error?

![image](https://github.com/Flegma/ebot-cs2-install-script/assets/39219862/e5c867da-7bc8-422c-8188-eb8b0072e3fe)

**Comment by Flegma** [comment]
This is probably a new version of the script from yesterday that you're using, right? 
Is it working now when running as as service or?

**Comment by xe1os** [comment]
> This is probably a new version of the script from yesterday that you're using, right? Is it working now when running as as service or?

Yes tried using your new script but it is still showing as Loading for both eBOT and WebSocket. 

SSL (certbot) is working.

Port forwarding details.

![image](https://github.com/Flegma/ebot-cs2-install-script/assets/39219862/8a4d627d-50de-41c1-8fb7-6b6ba6935939)

**Comment by xe1os** [comment]
@Flegma dockerised ebot is working fine with me but there is no SSL :(

**Comment by Flegma** [comment]
Im not sure whats your exact setup, can you explain what are you trying to achieve? eBot on a public server with (sub)domain name, and with SSL certificate?

Can you check if you can change http:// to https:// in the config (LOG_ADDRESS_SERVER) and check?
If you try the new script, what is "service ebot-cs2-app status" and "service ebot-cs2-logs status" output giving you?

**Comment by xe1os** [comment]
> Im not sure whats your exact setup, can you explain what are you trying to achieve? eBot on a public server with (sub)domain name, and with SSL certificate?

Yes

> 
> Can you check if you can change http:// to https:// in the config (LOG_ADDRESS_SERVER) and check? If you try the new script, what is "service ebot-cs2-app status" and "service ebot-cs2-logs status" output giving you?

![image](https://github.com/Flegma/ebot-cs2-install-script/assets/39219862/b8289046-390f-496e-b77b-cb33e77b9be7)
![image](https://github.com/Flegma/ebot-cs2-install-script/assets/39219862/a2c42747-5967-454b-b55a-22a9f9531276)

**Comment by Flegma** [comment]
I will need to check it out and test, i've never used it with SSL before, it's probably some configuration change that needs to be added when using SSL cert. 
Can you open port 12361 as well, also?

**Comment by xe1os** [comment]
> I will need to check it out and test, i've never used it with SSL before, it's probably some configuration change that needs to be added when using SSL cert. Can you open port 12361 as well, also?

Tried opening port 12361. Still no luck

**Comment by xe1os** [comment]
ebot-cs2-app config.ini

![image](https://github.com/Flegma/ebot-cs2-install-script/assets/39219862/5ea9e1fa-1d66-40c6-bcf8-9abef896bab4)
